### PR TITLE
include the height of a block in the precomputed block filename

### PR DESCRIPTION
**Pull request by deepthiskumar**
Makes it easier to look for missing blocks at specific heights

### CODE REVIEW

File: src/lib/mina_lib/coda_subscriptions.ml [hunk 1]
```diff
@@ -156,7 +156,14 @@ let create ~logger ~constraint_constants ~wallets ~new_blocks
...
              | Some _, Some network, Some bucket ->
                  let hash_string = State_hash.to_string hash in
-                 let name = sprintf "%s-%s.json" network hash_string in
+                 let height =
+                   Mina_transition.External_transition.Validated
+                   .blockchain_length new_block
+                   |> Mina_numbers.Length.to_string
+                 in
+                 let name =
+                   sprintf "%s-%s-%s.json" network height hash_string
```
Comment (psteckler) [inline src/lib/mina_lib/coda_subscriptions.ml:165]: would it be too verbose to use `net:%s-height:%s-hash:%s` ?

**Review by psteckler — APPROVED** [review]

**Review by mrmr1993 — APPROVED** [review]

File: src/lib/mina_lib/coda_subscriptions.ml (hunk 1 above)
Comment (mrmr1993) [reply src/lib/mina_lib/coda_subscriptions.ml:165]: I would worry about using `:`, since it makes the filename invalid for windows.

**Comment by aneesharaines** [comment]
!approved-for-mainnet

**Comment by mrmr1993** [comment]
@deepthiskumar before I merge this, can you confirm that there aren't corresponding infra changes that need to happen if we change the filename?

**Comment by deepthiskumar** [comment]
> @deepthiskumar before I merge this, can you confirm that there aren't corresponding infra changes that need to happen if we change the filename?

Yes, the network name passed using cli flag is still used the same way. I could not find any tooling around this except for the watchdog metric and that does not involve the filename

**Pull request merged**